render.js

```diff
@@ -0,0 +1,16 @@
+var render = (function() {
+
+  var MovieInfo = function(response) {
```

Comment:
The function name "MovieInfo" should start with  a lower case character. Try changing it to "movieInfo"